actionview/lib/action_view/view_paths.rb

```diff
@@ -85,8 +83,8 @@ def view_paths
       # Set the view paths.
       #
       # ==== Parameters
-      # * <tt>paths</tt> - If a PathSet is provided, use that;
-      #   otherwise, process the parameter into a PathSet.
+      # * <tt>paths</tt> - If an Array of String is provided, it gets converted into
```

Comment:
Ah, the other documentation has it capitalized so :+1:, but add an s to String still.